Items/08_Tool_Sets.txt

```diff
@@ -0,0 +1,215 @@
+their weight in gold. These tools keep blades sharp, rust-free and
+ready for action.
+
+Relevant Skill(s): Weapons - Melee, Weapons - Thrown
+Context: Bladed Weapons
+Noise Level: Low
+Weight: 
+
+Digital Artist's Program and Tablet Package ($250):
+	A pressure-sensitive digital tablet and stylus complete with 
+TIR Digital Painting, which can be used for drawing, painting, and 
+editing photos and documents. Comes with a protective case and 
+rechargeable 8-hour battery.
+
+Relevant Skill(s): Performance, Deception
+Context: Digital Art, Imagery
+Noise Level: Low
+Weight:
+
+Destructive Entry Kit ($200):
+	Includes several tools useful for making an irreversible 
+entrance into secured locations, such as bolt cutters, prybars, 
+a small, concealable drill, core puller, core shearer, and hinge 
+pin striker.
+
+Relevant Skill(s): Sleight of Hand, Athletics
+Context: Mesh fencing, doorways, windows
+Noise Level: Low-High
+Weight: 
+
+Disguise Kit ($200):
+	Includes a selection of make-up and application tools, hair 
+styling necessities, items for a quick outfit change including a variety
+of hats, scarves, sunglasses and flashy jewelery, several sets of
+personalized color contacts, and temporary hair dye in a variety of colors.
+
+Relevant Skill(s): Deception
+Context: Broad
+Noise Level: Low
+Weight: 
+
+Electrician's Kit ($350):
+	Voltage testers; sensors; insulated wire strippers, ratchets, 
+screwdrivers, and pliers; reels of wire; soldering supplies; breadboards; 
+resistors; capacitors; inductors; rechargeable temporary power supplies; 
+and a miniaturized 3D circuit printer allow the professional electrician 
+to maintain, repair, and construct all manner of electronic gadgetry.
+
+Relevant Skill(s): Repair, Jury-rigging
+Context: Electronics, Electrical Components, Circuitboards. Repair, 
+	construction, repurposing.
+Noise Level: Low
+Weight:
+
+Electronic Lock Spoofing Kit ($250):
+	[ph]
+
+Relevant Skill(s): Hacking
+Context: Electronic and/or computerized locking objects
+Noise Level: Low
+Weight: 
+
+Firearm Maintenance and Care Kit ($100):
+	[ph]
+
+Relevant Skill(s): Weapons - [All firearms categories]
+Context: Cleaning, Servicing
+Noise Level: Low
+Weight: 
+
+Forger's Implements ($250):
+	[ph]
+
+Relevant Skill(s): Deception, Sleight of Hand
+Context: Forgery (Eg. Documents, ID Cards, etc. Not passwords)
+Noise Level: Low
+Weight: 
+
+General Research Analysis and Cross-referencing AI ($350):
+	A portable hand terminal containing a massive solid-state data drive 
+and a smaller drive with a non-sentient AI specialized in data retrieval. 
+The AI can help investigate inquiries by intelligently loading up reputable 
+documents and providing summaries. Performance greatly improved by access to 
+academic or library databanks. Takes time to achieve best results. Comes with 
+a protective case and rechargeable 8-hour battery.
+
+Relevant Skill(s): Knowledge - All, Investigation
```

Comment:
I'm really torn on this item. On the one hand, it seems OP. On the other hand, it's got a price tag to match and wikipedia is a real thing.

Reply:
It's also entirely controlled by the GM and basically useless in combat.